src/components/pages/pages.stories.js

```diff
@@ -111,6 +111,7 @@ function makeStory(name, themeSelector) {
           <DialogFullScreen
             open={ store.get('open') }
             onCancel={ handleCancel }
+            showCloseIcon
```

Comment:
Adding this would indicate that we have made a breaking change somewhere.

Reply:
I am discarding this commit. that entire commit I think is left over by accident from early version where I did breaking changes and I wanted to add knob to pages for and few other components to control if close icon should be shown or not.